docs/source/models/gbd2017/causes/lower_respiratory_infections/index.rst

```diff
@@ -12,17 +12,33 @@ and bronchiolitis, is a major global killer of both children and adults. Symptom
 include shortness of breath, weakness, fever, coughing and fatigue. It is important to check for a fever. Symptoms can last about 7 days and the infection is contagious
 to others shortly before and while experiencing symptoms. It is mainly
 caused by four microorganisms - Streptococcus pneumoniae (*pneumococcal
-pneumonia*), Haemophilus influenzae type B (*Hib*),influenza, and respiratory
+pneumonia*), Haemophilus influenzae type B (*Hib*), influenza, and respiratory
 syncytial virus (*RCV*). LRI can be caused by multiple pathogens and the pathogens may co-infect.
```

Comment:
Is there a difference between being caused by multiple pathogens, and pathogens co-infecting?

Reply:
My understanding when I read this was that "can be caused by multiple pathogens" means that any one of the listed pathogens can cause LRI, and that "pathogens may co-infect" means that someone who has LRI may be infected by more than one of the pathogens simultaneously. Does that sound accurate, @XiaoluQianUW ?

Perhaps it could be reworded to be clearer? Any suggestions, @beatrixkh ?

Reply:
I might have not use the best wording for this, but that sound accurate. I will try to reword this to make it clearer.